app/views/api/clidocs/casestudy_metagenomics.html.erb

```diff
@@ -1,121 +1,434 @@
+<!--   -->>read2/1|2R
+<!--   -->0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
+<!--   -->>read2/1|3R
+<!--   -->0 0 0 0 393310 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
+<!--   -->>read2/2|1
+<!--   -->0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
+<!--   -->>read2/2|2
+<!--   -->1534 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
+<!--   -->>read2/2|3
+<!--   -->0 0 0 0 0 0 0 0 0 0 0 0 1370023 0 0 0 0 0 0 0 0 0 0 0
+<!--   -->>read2/2|1R
+<!--   -->0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 488447 0 0 0 0 0 0 0
+<!--   -->>read2/2|2R
+<!--   -->0 0 0 0 0 0 33882 4930 4930 0 392734 1 392734 392734 392734 392734 392734 392734 392734 392734 392733 392733 392733 392733 392734
+<!--   -->>read2/2|3R
+<!--   -->0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
+<!--   -->>read3/1|1
+<!--   -->0 0 0 0 0 0 0 0 0 0 1 0 0 0 0 1754192 0 6282 0 7164 0 0 0 0 0
+<!--   -->>read3/1|2
+<!--   -->0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
+<!--   -->>read3/1|3
+<!--   -->332163 332163 332163 2 1 332163 2 332163 332163 332163 332163 332163 1 332163 332163 332163 332163 332163 332163 332163 332163 332163 0 0
+<!--   -->>read3/1|1R
+<!--   -->0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 322865 7868 0 0 0 0 0 0
+<!--   -->>read3/1|2R
+<!--   -->0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
+<!--   -->>read3/1|3R
+<!--   -->0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 1 0 0 0 0 0 0 0 0
+<!--   -->>read3/2|1
+<!--   -->0 0 0 0 0 0 0 0 0 0 632569 0 0 0 0 0 0 0 0 0 0 0 0 0 0
+<!--   -->>read3/2|2
+<!--   -->0 0 0 8839 0 0 40754 0 0 0 0 323415 0 286 1 1760 0 0 0 0 0 0 0 0 0
+<!--   -->>read3/2|3
+<!--   -->0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 28216 0 0 0 0 0 0 0
+<!--   -->>read3/2|1R
+<!--   -->332163 2 2 332163 332163 332163 332163 332163 332163 332163 1 332163 332163 332163 2 2 1 332163 332163 0 0 0 0 0 0
+<!--   -->>read3/2|2R
+<!--   -->67819 0 0 0 5855 1 1 2 519963 5206 0 1245748 0 0 0 0 0 0 0 0 0 101127 0 0 0
+<!--   -->>read3/2|3R
+<!--   -->0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 13687 0 0 0
+        </pre>
         <p>
-          As the analysis only reports the taxon IDs, we now lookup the names and ranks of all taxa found in the sample <i>(timing: 1.3 secs)</i>.
-    </p>
+          This command will look up each 9-mer in the input data
+          and output the associated taxon if any is found.
+          With the <code>-o</code> or <code>--one-on-one</code> flag,
+          the unrecognized 9-mers will be included as a "0" taxon,
+          the use of which will be clarified in the next step.
+          Note: the real output would contain newlines between taxa,
+          which have been replaced here by spaces for readability.
+        </p>
+      </div>
+    </div>
+
+    <div class="card">
+      <div class="card-title card-title-colored"><h2 class="card-title-text">(Optional) Filtering</h2></div>
+      <div class="card-supporting-text">
+        <p>
+          Especially when running the pipeline with all six frames,
+          a lot of the translations are invalid.
```

Comment:
are invalid since we only expect one of the reading frames to yield a valid protein.

_[Original comment](https://github.ugent.be/unipept/unipept/pull/642#discussion_r16555) by @bmesuere on Mon Aug 20 2018 at 13:59._